runtime/doc/options.txt

```diff
@@ -5292,9 +5292,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	unescaping, so to keep yourself sane use |:let-&| like shown above.
 							*shell-powershell*
 	To use PowerShell: >
-		let &shell = has('win32') ? 'powershell' : 'pwsh'
+		let &shell = executable('pwsh') ? 'pwsh' : 'powershell'
 		let &shellcmdflag = '-NoLogo -NoProfile -ExecutionPolicy RemoteSigned -Command [Console]::InputEncoding=[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;'
-		let &shellredir = '2>&1 | Out-File -Encoding UTF8 %s; exit $LastExitCode'
+		let &shellredir = '-RedirectStandardOutput %s -NoNewWindow -Wait'
```

Comment:
Since `exit $LastExitCode` was removed, will Nvim correctly get the exit code after this change?

Reply:
I don't know that. But running commands with wrong arguments in cmd directly gives the following results:

```cmd
C:\Users\ACER> pwsh -c "sldkfj  2>&1 | Out-File -Encoding UTF8 err.txt; exit $LastExitCode"
sldkfj: The term 'sldkfj' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

C:\Users\ACER> echo Exit Code is %errorlevel%
Exit Code is 0

C:\Users\ACER> pwsh -c "sldkfj  2>&1 | Out-File -Encoding UTF8 err.txt"
sldkfj: The term 'sldkfj' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

C:\Users\ACER> echo Exit Code is %errorlevel%
Exit Code is 1

C:\Users\ACER> pwsh -c "Start-Process sort -RedirectStandardInput sldkfj.txt -RedirectStandardOutput out.txt -NoNewWindow -Wait"
Start-Process: This command cannot be run because either the parameter "RedirectStandardInput 'C:\Users\ACER\sldkfj.txt'" has a value that is not valid or cannot be used with this command. Give a valid input and Run your command again.

C:\Users\ACER> echo Exit Code is %errorlevel%
Exit Code is 1

C:\Users\ACER> pwsh -c "Start-Process sort -RedirectStandardInput sldkfj.txt -RedirectStandardOutput out.txt -NoNewWindow -Wait; exit $LastExitCode"
Start-Process: This command cannot be run because either the parameter "RedirectStandardInput 'C:\Users\ACER\sldkfj.txt'" has a value that is not valid or cannot be used with this command. Give a valid input and Run your command again.

C:\Users\ACER> echo Exit Code is %errorlevel%
Exit Code is 0
```

So, `; exit $LastExitCode` spends (?) the exit code? I don't know how Nvim handles that. I'm sure you can understand whether `; exit $LastExitCode` is needed or not from the above examples.